base/src/main/kotlin/net/corda/v5/base/annotations/VisibleForTesting.kt

```diff
@@ -1,6 +1,10 @@
 package net.corda.v5.base.annotations
 
-/** The annotated object would have a more restricted visibility were it not needed in tests. */
+/** The annotated object would have a more restricted visibility were it not needed in tests.
+ *
+ * This annotation has technically no effect on the code - it is purely a marker that the visibility of
+ * the annotated entity is chosen for testing purposes.
```

Comment:
"This annotation has technically no effect on the code. It is only a marker to indicate the visibility of the annotated entity for testing purposes." ?